app/src/main/java/com/jerboa/datatypes/types/Others.kt

```diff
@@ -382,6 +382,22 @@ enum class ModlogActionType {
     AdminPurgeComment,
 }
 
+/**
+ * Different show nsfw states
+ */
+enum class ShowNsfwTypes {
```

Comment:
Move this to Feat.kt remove serialize stuff l. That is only needed for custom stuff we get from from the server. This exists only client side thus doesn't need it. And preferable if possible make it Parcelable